test/desktop/arduino_interface_test.cc

```diff
@@ -0,0 +1,119 @@
+// Copyright 2017 Google Inc.
```

Comment:
Would you be able to "git copy" this file from the original? That would help me to spot the changes that were made to it.

Reply:
My "original" file was https://github.com/mum4k/arduino_abstractions/blob/72db8fb88902fabba7206b236b67e354edc7627e/arduino/arduino_interface_test.cc from https://github.com/mum4k/arduino_abstractions/pull/5. 
How can we do it without losing the changes made by PR#5?

Reply:
Fair point, trying to tie the files together now may be too complicated and not worth it. I am happy to review it as it is.